lib/processors/themeBuilderWorker.js

```diff
@@ -92,3 +72,129 @@ if (!workerpool.isMainThread) {
 		execThemeBuild
 	});
 }
+
+
+export class FsMainThreadInterface {
+	#comPort = null;
+	#fsInterfaceReader = null;
+	#cache = Object.create(null);
```

Comment:
Maybe I'm missing something but isn't the relationship from FsMainThreadInterface to FsWorkerThreadInterface 1:1?

So whatever is requested in the worker will also be cached there (which I think makes sense, but this cache in the main thread will never get any hits?

It might make sense to have an `@ui5/fs` level cache for the combo reader that is shared by all FaMainThreadInterface instances, since many worker will probably access the same resources and they are not expected to change while the task is running.

Reply:
Indeed, that was my point initially. 
I'll try to make the `FaMainThreadInterface` a singleton that manages all the connections, but keeps a cache for the already resolved resources, because that's the biggest possible improvement here. Each `Resource` gets resolved by the combo reader's globbing for up to 40ms. Caching in that case for big builds like `themebuild_sap_belize` might squize some more milliseconds from the build step.